task3/Task3/menu.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Drawing;
+
+namespace Task3
+{
+    class Meal
+    {
+        private string name;
+        private double price;
+        private string pathToImage;
+
+        public string Name
+        {
+            get { return name; }
+            set { name = value; }
+        }
+
+        public double Price
+        {
+            get { return price; }
+            set { price = value; }
+        }
+
+        public string PathToImage
+        {
+            get { return pathToImage; }
+            set { pathToImage = value; }
+        }
+
+        Meal() { }
```

Comment:
Remove constructor, replace it with `Meal(string name, double price)`
I guess we can have a meal without an image.

Also, add name and price validation. The price must be a positive number and the name cannot be null